src/style.css

```diff
@@ -0,0 +1,83 @@
+#container {
+  left: 30%;
+  position: relative;
+  top: 50px;
+  width: 30%;
+  border: 1px solid wheat;
+  padding: 20px 20px 0 20px;
+}
+
+.title {
+  display: block;
+  font-size: 24px;
+}
+
+.refresh {
+  float: right;
+  position: relative;
+  top: 3px;
+}
+
+.inputText {
+  margin-top: 15px;
+  width: 108%;
+  height: 40px;
+  position: relative;
+  left: -20px;
+  font-size: 18px;
+  border: 1px solid wheat;
+  border-left: none;
+  border-right: none;
+
+  /* border-style: hidden; */
+}
+
+::placeholder {
+  font-style: italic;
+  position: relative;
+  left: 15px;
+}
+
+.todo-item {
+  text-decoration: none;
+
+  /* background: blue; */
```

Comment:
- [ ] it would be better to remove the unused commented code in order to keep the file clean and pro

Reply:
This change has been implemented please refer to the screen shot below.
![image](https://user-images.githubusercontent.com/117091923/222160106-9e372d1f-1dd5-43da-a6e2-dcd915bbab7c.png)